packages/pluggableWidgets/gallery-text-filter-native/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "@mendix/pluggable-widgets-tools/configs/tsconfig.base",
+    "baseUrl": "./",
+    "include": ["node_modules/@mendix/pluggable-widgets-tools/typings", "./src"],
```

Comment:
this must include ./typings as well, otherwise it will not allow the usage of typings auto generated